homeassistant/components/weatherkit/strings.json

```
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "WeatherKit Setup",
+        "description": "Enter your location details and WeatherKit authentication credentials below.",
+        "data": {
+          "name": "Name",
+          "location": "Location",
+          "key_id": "Key ID",
+          "team_id": "Apple Team ID",
```

```suggestion
          "team_id": "Apple team ID",
```
Only if "Apple Team" is a brand, otherwise ignore